compiler/ir/ir.tree/src/org/jetbrains/kotlin/ir/util/DeepCopyIrTreeWithSymbols.kt

```diff
@@ -65,6 +65,14 @@ open class DeepCopyIrTreeWithSymbols(
     }
 
     protected fun mapDeclarationOrigin(origin: IrDeclarationOrigin) = origin
+    protected fun mapDeclarationOrigin(declaration: IrDeclaration):IrDeclarationOrigin {
+        return when {
+            declaration.origin == IrDeclarationOrigin.LOCAL_FUNCTION_FOR_LAMBDA && (declaration is IrFunction) && !declaration.isInline -> {
```

Comment:
Do really such functions exist - with origin = LOCAL_FUNCTION_FOR_LAMBDA and inline?

Reply:
Hm... good question. will investigate it.